src/Controller/RefileRequestController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+namespace NYPL\Services\Controller;
+
+use NYPL\Services\ItemClient;
+use NYPL\Services\Model\RefileRequest\RefileRequest;
+use NYPL\Services\Model\Response\RefileRequestResponse;
+use NYPL\Services\ServiceController;
+use NYPL\Services\SIP2Client;
+use NYPL\Starter\APIException;
+use NYPL\Starter\APILogger;
+use NYPL\Starter\Model\Response\ErrorResponse;
+use Slim\Http\Response;
+
+/**
+ * Class RefileRequestController
+ *
+ * @package NYPL\Services\Controller
+ */
+class RefileRequestController extends ServiceController
+{
+
+    /**
+     * @SWG\Post(
+     *     path="/v0.1/recap/refile-requests",
+     *     summary="Process a refile request",
+     *     tags={"recap"},
+     *     operationId="createRefileRequest",
+     *     consumes={"application/json"},
+     *     produces={"application/json"},
+     *     @SWG\Parameter(
+     *         name="NewRefileRequest",
+     *         in="body",
+     *         description="Request object based on the included data model",
+     *         required=true,
+     *         @SWG\Schema(ref="#/definitions/NewRefileRequest")
+     *     ),
+     *     @SWG\Response(
+     *         response=200,
+     *         description="Successful operation",
+     *         @SWG\Schema(ref="#/definitions/RefileRequestResponse")
+     *     ),
+     *     @SWG\Response(
+     *         response="401",
+     *         description="Unauthorized"
+     *     ),
+     *     @SWG\Response(
+     *         response="404",
+     *         description="Not found",
+     *         @SWG\Schema(ref="#/definitions/RefileRequestErrorResponse")
+     *     ),
+     *     @SWG\Response(
+     *         response="500",
+     *         description="Generic server error",
+     *         @SWG\Schema(ref="#/definitions/RefileRequestErrorResponse")
+     *     ),
+     *     security={
+     *         {
+     *             "api_auth": {"openid offline_access api write:hold_request readwrite:hold_request"}
```

Comment:
Would the api_auth scopes change from `write:hold_request readwrite:hold_request` to `readwrite:refile_request` ?

Reply:
We can talk to Kevin about this, but I think we can just default to the hold_request scope for all ReCAP endpoints since they're the only ones using this service at present. We may want to change it down the line, but I believe all of their endpoints are now sending the hold_request scope.